src/Alert/index.js

```diff
@@ -44,7 +44,7 @@ const Alert = ({ header, variant, children }) => {
       <div className='spectrum-Icon spectrum-UIIcon-InfoLarge spectrum-Alert-icon'>
         {icon}
       </div>
-      <div className='spectrum-Alert-header'>{header}</div>
+      {header && <div className='spectrum-Alert-header'>{header}</div>}
```

Comment:
What does this line do?

Reply:
Nevermind. I figured it out, it's conditional rendering.

Reply:
If you don't provide a header then it doesn't add the div and the spacing:

<img width="383" alt="Screenshot 2020-07-21 13 31 16" src="https://user-images.githubusercontent.com/353180/88087273-a55a3580-cb56-11ea-8435-574764eb7923.png">

otherwise you get a lot of whitespace at the top.

<img width="380" alt="Screen Shot 2020-07-21 at 1 33 55 PM" src="https://user-images.githubusercontent.com/353180/88087409-d6d30100-cb56-11ea-8b4d-b468b31af434.png">